rules-topics/09.00_agility-abilities.md

```diff
@@ -9,254 +9,187 @@ The Agility Point cost of each ability is equal to its level.  Characters may "m
+| **Verbal** | "Physical Waylay" |
+| **Description** | This skill is a strike from behind that renders the target unconscious and helpless for 20 seconds. A Waylay strike is dealt to the back of the target, not to the head, with the striking surface of the character's weapon. A character skilled in Waylay can refresh the effect of a waylay without expending a use of the skill by tapping the target on the back and saying, "Refresh Waylay". This effect may be affected by the Transfer Malady or Cure Body spells. A waylaid target will awaken instantly if damaged or healed.  This is a SLEEP effect. |
 
-
-### Level 7 Agility Abilities
-#### **Variable Critical **
-| Ability | **Variable Critical** |
+| | **Critical Empowerment** |
 |:---|---|
-| **Type**| Agility|
-| **Level**| 7 |
+| **Level** | 7 |
 | **Duration** | Instant |
-| **Call**| "Variable Critical &lt;Effect>" |
-| **Description** | This skill allows user to swing one other agility skill effect against a single target. Effect ends at rest or when invoking Backstab against a different target. The other skill it is being combined with is used at the same time this skill is.|
+| **Verbal** | "I empower my \<effect> strike!" |
+| **Description** | This skill allows user to swing one other agility skill effect against a single target. Effect ends at rest or when invoking Backstab against a different target. The other skill it is being combined with is used at the same time this skill is. |
 
-#### **Warbow Training**
-| Ability | **Warbow Training** |
+| | **Warbow Training** |
 |:---|---|
-| **Type**| Agility|
-| **Level**| 7 |
+| **Level** | 7 |
 | **Duration** | Rest |
-| **Call**| "My Bow is Ready" |
-| **Description** |This skill allows the user to block with their bow or crossbow without requiring it to be restrung.  This lasts until rest.  User must spend 1 minute roleplaying inspecting their bow and recalling their training.|
-| **Requirements**| Ranged Weapon|
-
+| **Verbal** | "My Bow is Ready" |
+| **Description** | This skill allows the user to block with their bow or crossbow without requiring it to be restrung.  This lasts until rest.  User must spend 1 minute roleplaying inspecting their bow and recalling their training. |
 
-### Level 8 Agility Abilities
-#### **Stun**
-| Ability | **Stun**   |
+| | **Stunning Strike ** |
 |:---|---|
-| **Type**| Agility |
-| **Level**| 8 |
+| **Level** | 8 |
 | **Duration** | 20 Seconds |
-| **Call**| "Physical Stun"  |
+| **Verbal** | "Physical Stun" |
 | **Description** | This skill is a strike that renders the target stunned for 20 seconds.  Stunned target may not move, communicate in any manner, activate or use items, or use in-game skills.  This effect may be affected by the Transfer Malady or Cure Body. The effect of this skill may not be stacked or extended. This is a STUN effect. |
 
```

Comment:
Surprising Proficiency is missing here.

Reply:
Only needs to be added to the agility json